### PR TITLE
First Commits

### PR DESCRIPTION
**Pull request by felddy**
#  #

## 🗣 Description ##

Copy text from https://www.cisa.gov/BadPractices into repository.

## 💭 Motivation and context ##

The text is required for contributors to both read and suggest modifications. 

## 🧪 Testing ##

Automated tests locally and in GitHub Actions. 

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
⚠️ 👍

**Review by jsf9k — APPROVED** [review]
Does it make sense to include the "Contributing" blurb here?  I know we want people to contribute, but it seems to muddy the waters a bit here.

File: README.md [hunk 1]
```diff
@@ -1,23 +1,45 @@
...
+   internet-accessible technologies.
+
+While these practices are dangerous for critical infrastructure and NCFs, CISA
+encourages all organizations to engage in the necessary actions and critical
+conversations to address bad practices.
+
+*Note: This list is focused and does not include every possible inadvisable
+cybersecurity practice. The lack of inclusion of any particular cybersecurity
+practice does not indicate that CISA endorses such a practice or deems such a
+practice to present acceptable levels of risk.*
```
Comment (mcdonnnj) [inline README.md:43]: Maybe it would be useful to add a link to the page this is copied from?

**Review by mcdonnnj — APPROVED** [review]
:shipit: Just a suggestion for consideration.

**Comment by felddy** [comment]
> Does it make sense to include the "Contributing" blurb here? I know we want people to contribute, but it seems to muddy the waters a bit here.

I agree the contributing blurb doesn't fit as well with this type of project, but it does serve as a placeholder.   I think we can let the new maintainers decide to change it.   They're going to want to guide people to the ["Discussions"](https://github.com/cisagov/bad-practices/discussions) tab.  This might be a good place to do that.

File: README.md (hunk 1 above)
Comment (felddy) [reply README.md:43]: The link is at the top of the page in the about block of the repo: 
image

Comment (mcdonnnj) [reply README.md:43]: Ah, that makes sense.

**Pull request merged**